plugins/org.eclipse.elk.core.service/src/org/eclipse/elk/core/service/IDiagramLayoutConnector.java

```diff
@@ -14,25 +14,16 @@
 import org.eclipse.ui.IWorkbenchPart;
 
 /**
- * Interface for managers of diagram layout. A diagram layout manager is responsible for transforming
+ * Interface for connectors of diagram layout. A diagram layout connector is responsible for transforming
  * the diagram contained in a workbench part into a layout graph, which is an instance of the
  * KGraph meta model. Furthermore it must handle the transfer of concrete layout data from the
  * layout graph back to the diagram after a layout has been computed.
  * 
- * @param <T> the type of diagram part that is handled by this diagram layout manager
  * @kieler.design proposed by msp
  * @kieler.rating yellow 2012-07-19 review KI-20 by cds, jjc
  * @author msp
  */
-public interface IDiagramLayoutManager<T> {
-
-    /**
-     * Determine whether this layout manager is able to perform layout for the given object.
-     * 
-     * @param object a workbench part or edit part
-     * @return true if this layout manager supports the object
-     */
-    boolean supports(Object object);
```

Comment:
Is this method removed because everyone now supplies his own layout setups, so that ELK doesn't check anymore if it has anything that can layout the current diagram?